connect-inject/handler.go

```diff
@@ -393,6 +478,76 @@ func (h *Handler) defaultAnnotations(pod *corev1.Pod, patches *[]jsonpatch.JsonP
 	return nil
 }
 
+func (h *Handler) consulNamespace(ns string) string {
```

Comment:
can you add docs for this

Reply:
Updated and pulled out the first comment.